Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
  * SMTChecker: Report out of bounds index access for arrays and fixed bytes.
  * Standard JSON: Model checker option ``settings.modelChecker.targets`` also accepts ``outOfBounds``.
  * Yul Optimizer: Added a new step FunctionSpecializer, that specializes a function with its literal arguments.
+ * References Resolver: Detect circular references to other contracts across libraries and free functions
```

Comment:
It is not really the references resolver. Cannot think of a better word now. We can just call it "Analysis" if we do not find a better word.

Also please use "code references". References is too general.